pkgconfig/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# CMakeLists.txt for libyui-mga/pkgconfig
+#
+# Support for pkg-config:
+#
+# Generate a libyui.pc file from libyui.pc.in and install it to
+# /usr/lib64/pkgconfig.
+#
+# A .pc file specifies how to use a development package, in particular linker
+# flags (-lyui-gtk), compiler flags (including include directories) and paths.
+# See man pkg-config.
+#
+# Notice that libyui-mga linking is needed for libyui-mga-* plugins 
```

Comment:
NP: Copy&paste leftover: Should be libyui-gtk

Reply:
changed, thanks